addons/common/fnc_uniqueUnitItems.sqf

```diff
@@ -0,0 +1,46 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_uniqueUnitItems
+
+Description:
+    Retrievs a unique list of items in the units inventory.
+
+Parameters:
+    _unit           - Unit to retrieve the items from
+    _weaponItems    - Include weapons, attachments, loaded magazines (Default: false)
+    _backpack       - Include items in backpack (Default: true)
+    _vest           - Include items in vest     (Default: true)
+    _uniform        - Include items in uniform  (Default: true)
+
+Example:
+    (begin example)
+    _allItems = [player, true, false] call CBA_fnc_uniqueUnitItems
+    (end)
+
+Returns:
+    Array of item classnames <ARRAY>
+
+Author:
+    Dedmen
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+SCRIPT(uniqueUnitItems);
+
+params [["_unit", objNull, [objNull]], ["_weaponItems", true, [true]], ["_backpack", true, [true]], ["_vest", true, [true]], ["_uniform", true, [true]]];
+
+private _allItems = (assignedItems _unit);
+if (_uniform) then {_allItems append ((getItemCargo (uniformContainer _unit)) select 0);};
+if (_vest) then {_allItems append ((getItemCargo (vestContainer _unit)) select 0);};
+if (_backpack) then {_allItems append ((getItemCargo (backpackContainer _unit)) select 0);};
```

Comment:
parenthesis aaaaa

Reply:
Perfectly fine here to quickly see the order they are executed in.

Reply:
aaaaa

Reply:
```sqf
getItemCargo uniformContainer _unit select 0
```
this is just unreadable IMO.